SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.h

```diff
@@ -59,6 +59,7 @@ class Phase2TrackerDigitizerAlgorithm {
                                   const edm::ParameterSet& conf_specific,
                                   edm::ConsumesCollector iC);
   virtual ~Phase2TrackerDigitizerAlgorithm();
+  virtual void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
```

Comment:
The `fillDescriptions()` and `fillPSetDescription()` functions must be `static`. This class looks also like a helper class, so `fillPSetDescription()` would align better with the goal of (eventually) implementing proper `fillDescriptions()` for MixingModule.